ChechpointStyles.css

```diff
@@ -0,0 +1,78 @@
+
+body { background-color: silver;
+
+}
+
+
+
+nav { display: block;
+     padding: 40px;
+     background-color: green;
+     
+}
+
+a {color: antiquewhite;
+   font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-weight: 700;
+}
+
+.float-left{ float: left;
+display: block;
+padding: 8px;
+border-right: 1px solid whitesmoke;
+text-align: center;
+}
+
+.float-right{ float: right;
+display: block;
+padding: 8px;
+border-right: 1px solid whitesmoke;
+}
+
+h1 { text-align: center;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+text-decoration: underline;}
+
+#IMAGE { text-align: center;
```

Comment:
Try to use `kebab-case` for naming in CSS. Refer to the guide I mentioned in a comment above.